docs/setup.md

```diff
@@ -3,22 +3,26 @@
 
 # Installation
 1. To install a widget, click **Content Hub** > **Discover**.
-2. From the list of widgets that appears, search for and select **SOC Management** widget.
-3. Click the **SOC Management** widget card.
+2. Search for the **Record Summary Card** widget.
+3. Click the **Record Summary Card** widget card.
 4. Click **Install** on the lower part of the screen to begin installation.
 
 # Configuration
 
 The following sections lay out information necessary to customize this widget.
 
-| Fields          | Description                                                                                                                     |
-|-----------------|---------------------------------------------------------------------------------------------------------------------------------|
-| Title           | Specify the heading for Record Summary Tile.               |
-| Data Source     | Select the Module which has JSON data.                     |
-| Filter  Condition | Filter Record Which Contains The JSON Data. It is recommended to apply filter, that filters out 1 record, i.e. by a unique field.                   |
-| Select Field     | Select a field from the module which has the JSON data.|
-|Enter the key of object to be rendered. | Specify the key under which the entire data is present, if the JSON data is already in the required format in the record, then leave the key field empty |
+| Fields                                  | Description                                                                                                                         |
+|-----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| Title                                   | Specify a title for the record summary card.                                                                                        |
+| Data Source                             | Select the module containing JSON data.                                                                                             |
+| Select Field                            | Select a field from the module that contains the data in JSON format.                                                               |
+| Filter  Condition                       | Filter records containing the JSON data. It is recommended to apply filters that return only 1 record i.e. by using a unique field. |
```

Comment:
Not recommended for the Record-Summary-Card, rather if there are multiple records there will be multiple cards